scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts

```diff
@@ -9,6 +10,7 @@ interface IScope extends ng.IScope {
     formattingOptions: Array<string>;
     schemaKeysOrdering: any;
     schemaKeysDisabled: any;
+    canValidateChars: boolean;
```

Comment:
I suggest calling the variable `characterValidationEnabled`. It's technically possible to validate characters even if it's not enabled so it's clearer to refer to it being enabled rather than being possible in the variable name.